eng/scripts/spell-check-public-api.ps1

```diff
@@ -48,13 +48,13 @@ foreach ($serviceDirectory in $directoresToScan) {
   $packageDirectories += Get-ChildItem -Path "$serviceDirectory/*/review" -Directory
 }
 
-$scanGlobs = @()
+$filePaths = @()
 foreach ($directory in $packageDirectories) {
-  $scanGlobs += "$directory/**/*.md"
+  $filePaths += (Get-ChildItem -Path $directory -Filter *.md -Recurse).FullName
```

Comment:
Does all *.md files equate to public api? It seems like it would include readme.md and other things as well.